frontend/src/components/Eligibility/EligibilityPage.js

```diff
@@ -96,8 +96,9 @@ const EligibilityPage = (props) => {
         <button
           type="button"
           className="button"
+          disabled={keycloaks.bcsc.authenticated}
           onClick={() =>
-            keycloak.login({
+            keycloaks.bceid.login({
```

Comment:
It might be worth it to just get rid of this IDIR button now, I think it just confuses people.

Reply:
Sure!